janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -6394,4 +6394,117 @@ public void testExistsMatchingIndex() {
         // test with invalid filter
         assertEquals(false, IndexSelectionUtil.existsMatchingIndex(conditions, null));
     }
+    
+    @Test
+    public void testReindexingForEdgeIndex() throws InterruptedException, ExecutionException {
+    	clopen(option(LOG_SEND_DELAY, MANAGEMENT_LOG), Duration.ofMillis(0),
+                option(KCVSLog.LOG_READ_LAG_TIME, MANAGEMENT_LOG), Duration.ofMillis(50),
+                option(LOG_READ_INTERVAL, MANAGEMENT_LOG), Duration.ofMillis(250)
+        );
+    	
+    	String indexWithDirectionIn = "edgesByAssocKindIn";
+    	String indexwithDirectionOut = "edgesByAssocKindOut";
+    	String indexWithDirectionBoth = "edgesByAssocKindBoth";
+    	
+    	String propertyKeyForIn = "assocKindForIn";
+    	String propertyKeyForOut = "assocKindForOut";
+    	String propertyKeyForBoth = "assocKindForBoth";
+    	
+    	
+    	//Schema creation
+    	if(!mgmt.containsEdgeLabel("egLabel")){
+			mgmt.makeEdgeLabel("egLabel").multiplicity(Multiplicity.MULTI).make();
```

Comment:
You can do this:
`EdgeLabel edgeLabel = mgmt.makeEdgeLabel("egLabel").multiplicity(Multiplicity.MULTI).make();`
so that you don't need to call `getEdgeLabel` later. Similar for property keys.

Reply:
done.